src/backends/ilp-quoter/index.js

```diff
@@ -80,22 +81,32 @@ class ILPQuoter {
     const currencyPair = utils.getCurrencyPair(this.currencyWithLedgerPairs,
                                                params.source_ledger,
                                                params.destination_ledger)
-    let amount, type
+    let amount
+    let type
+    let ledger
     if (params.source_amount) {
       amount = params.source_amount
       type = 'source'
-    }
-    if (params.destination_amount) {
+      ledger = params.source_ledger
+    } else if (params.destination_amount) {
       amount = params.destination_amount
       type = 'destination'
+      ledger = params.destination_ledger
```

Comment:
Is this backwards? If you know the destination amount, then you want to round to the precision of the source ledger right?